docs/modules/ROOT/pages/traits.adoc

```diff
@@ -367,6 +352,42 @@ It's disabled by default.
 
 !===
 
+| container
+| All
+| Configure integration contianer.
+  +
+  +
+  It's disabled by default.
+
+[cols="m,"]
+!===
+
+! container.request-cpu
+! The minimum amount of CPU required.
+
+! container.request-memory
+! The minimum amount of memory required..
+
+! container.limit-cpu
+! The maximum amount of CPU required.
+
+! container.limit-memory
+! The maximum amount of memory required.
+
+! container.service-port
+! To configure a different port exposed by the service (default `80`).
```

Comment:
I find the phrasing a bit confusing. My understanding is that this parameter enables to configure under which service port the container port is to be exposed. At least my brain feels more confortable with the later definition 😉.

Reply:
this has blindly moved from the service trait so yes, misleading